helm-chart/values.yaml

```
@@ -364,56 +355,27 @@ grafana:
       enabled: false
 
 ingress:
+  ## @param ingress.enabled Enable ingress resource generation for CVAT
+  ##
   enabled: false
-
-# In case you need an ingress, write them manually in your my.values.eml, see example below:
-#  enabled: true
-#  annotations:
-#    traefik.ingress.kubernetes.io/router.entrypoints: web
-#    kubernetes.io/ingress.class: traefik
-#  hosts:
-#    - host: cvat.local
-#      paths:
-#        - path: /api
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /admin
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /static
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /django-rq
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path: /profiler
-#          pathType: "Prefix"
-#          service:
-#            name: backend-service
-#            port: 8080
-#        - path : /
-#          pathType: "Prefix"
-#          service:
-#            name: frontend-service
-#            port: 80
-#  tls:
-#    - hosts:
-#      - <your_domain>
-#    secretName: ingress-tls-cvat
+  ## @param ingress.hostname Default host for the ingress record
```

```suggestion
  ## @param ingress.hostname Host for the ingress resource
```

(It's not really the default host, it's the only host.)